adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -135,6 +136,26 @@ public Agency getAgencyById(final String agencyId) {
         return agencyPerId.get(agencyId);
     }
 
+    /**
+     * Return a collection of Agency objects representing all the rows from agency.txt
```

Comment:
Mention immutability of the returned collection

Reply:
Good point, see 02f200bd43e833c2b79b2775665dc84f797c9427